src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -14,6 +15,7 @@ namespace Unity.Mathematics.Geometry
     /// are very cheap to construct and perform overlap tests with them.
     /// </remarks>
     [System.Serializable]
+    [Il2CppEagerStaticClassConstruction]
     internal struct MinMaxAABB : IEquatable<MinMaxAABB>
```

Comment:
Does this type have a static constructor?

Reply:
I think a lot of types won't have static constructors but for me it would be easier to just blanket apply them to every type than to worry about whether or not it actually has a static constructor on it.